src/environment.cpp

```diff
@@ -426,13 +426,15 @@ bool ServerEnvironment::line_of_sight(v3f pos1, v3f pos2, float stepsize, v3s16
 	return true;
 }
 
-void ServerEnvironment::kickAllPlayers(const std::string &reason)
+void ServerEnvironment::kickAllPlayers(u8 reason,
```

Comment:
For debugging purposes and for type checking at compile time (where possible) I'd have to agree with nerzhul regarding this comment. (I.e. seeing PASSWORD_INCORRECT -- I just made that up for example purposes -- in the debugger is a bit more informative than 0x04 (or whatever PASSWORD_INCORRECT's integer value actually is)